src/test/java/com/epam/tc/hw2/WebDriverSetup.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.tc.hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+public abstract class WebDriverSetup {
+    protected WebDriver driver;
+    protected final int timeOutWaits = 10;
```

Comment:
If a variable is not in use in child classes and there is no need to share it, it would be good to make it private